symplyphysics/laws/astronomy/objects_altitude_in_upper_culmination.py

```diff
@@ -0,0 +1,45 @@
+from sympy import (Eq, solve)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+)
+
+# Description
+## In observational astronomy, culmination is the passage of a celestial object across the observer's local meridian.
+## An object's altitude in degrees at its upper culmination is equal to 90 minus the observer's latitude plus
+## the object's declination.
+## https://учисьучись.рф/materials/shkolnaya-programma/astronomy/vysotasvetilvkulminacii/
+
+## Law is: h = 90 - L + d, where
+## h - object's altitude,
+## L - latitude,
+## d - object's declination.
+
+altitude = Symbol("altitude", angle_type)
```

Comment:
Altitude is usually a length. I suggest to rename to `angular_altitude`.
And don't forget to change it in the file name.